Zigbee2MqttAssistant/Views/Home/Index.cshtml

```diff
@@ -96,7 +97,7 @@
                     }
                 </td>
                 <td>
-                    @foreach (var entity in device.Entities.OrderBy(e => e.Component))
+                    @foreach (var entity in device.Entities.Where(e => !excludedComponents.Contains(e.Component)).OrderBy(e => e.Component))
```

Comment:
I think we should make a case insensitive comparison here... what do you think?

Reply:
No, MQTT topics are case-sensitive, so update_available is not the same as UPDATE_AVAILABLE
Found it here
https://www.hivemq.com/blog/mqtt-essentials-part-5-mqtt-topics-best-practices/